be/src/exprs/table_function/java_udtf_function.h

```diff
@@ -28,7 +28,7 @@ class JavaUDTFFunction final : public TableFunction {
     Status init(const TFunction& fn, TableFunctionState** state) const override;
     Status prepare(TableFunctionState* state) const override;
     Status open(RuntimeState* runtime_state, TableFunctionState* state) const override;
-    std::pair<Columns, ColumnPtr> process(TableFunctionState* state, bool* eos) const override;
+    std::pair<Columns, UInt32Column::Ptr> process(TableFunctionState* state) const override;
```

Comment:
How does the caller know if the process is eos without the output param `bool *eos`.

Well, I see, through `state->status()`. Please ignore this.

Reply:
> How does the caller know if the process is eos without the output param `bool *eos`.
> 
> Well, I see, through `state->status()`. Please ignore this.

`state->processed_rows() >= input_chunk->num_rows()` means all the input rows have finished processing.